# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/mgt/gcc34.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gnome-terminal/vt.c.orig	2004-09-12 10:06:08.461071992 +0200
+++ ./gnome-terminal/vt.c	2004-09-12 10:08:09.527667072 +0200
@@ -782,6 +782,7 @@
       vt_up(vt);    
       break;
     default:
+	;
     }
 
 }
@@ -806,6 +807,7 @@
       vt_down(vt);
       break;
     default:
+	;
     }
 }
 
@@ -870,6 +872,7 @@
 	}
       break; 
     default:
+	;
     }
 }
 
@@ -956,6 +959,7 @@
 	} 
       break;
     default:
+	;
     }
     /*vt->cursorx = (vt->cursorx-1) & (~7);*/
 }
@@ -980,6 +984,7 @@
 	} 
      break; 
     default:
+	;
     } 
 }
 
@@ -2073,9 +2078,11 @@
 */	      
 	break;
 	default:
+		;
 	}
       break;
 	default:
+		;
     } 
   /* do nothing!*/
 }
